lib/synapse-plugins/appcenter-plugin.vala

```diff
@@ -51,7 +51,8 @@ namespace Synapse {
 
                 appinfo = AppInfo.get_default_for_type ("x-scheme-handler/appstream", false);
                 if (appinfo != null) {
-                    // TRANSLATORS: The first %s is what the user searched for, the second will be replaced with the localized name of AppCenter
+                    // TRANSLATORS: The first %s is what the user searched for,
```

Comment:
I'm not sure if you can do this or not. Can we confirm that the whole comment shows up after regenerating po files? (but not commit those files ofc, just confirm it happens)

Reply:
hm, i'm getting:
```
xgettext: error while opening "src/Settings.vala" for reading: No such file or directory
```
regardless of the newline. 

I'll get that sorted and test again :+1:

Reply:
Ha woops. Sounds like we haven't updated POTFILES

Reply:
@kgrubb That's because the file `src/Settings.vala` specified in `POTFILES` no longer exist. Removing that line in `POTFILES` should work (but not commit this change too).

Reply:
It seems like it successfully prints the content, just as a multiline instead of a single line. Here's the `slingshot.pot` result:
![Screenshot from 2019-09-17 21-56-21](https://user-images.githubusercontent.com/8471701/65100526-0ee94080-d996-11e9-9fe6-9ce4d7c5fe28.png)